main.go

```diff
@@ -66,10 +70,15 @@ BE ON THE LOOKOUT !
 		for {
 
 			var input string
-			if _, err := fmt.Scanf("%s\n", &input); err != nil {
 
+			char, key, err := keyboard.GetSingleKey()
+			if err != nil {
 				log.Println(err)
 			}
+			defer fmt.Printf("snap: %q\r\n", char)
```

Comment:
I don't understand why you have this defer-ed print? What do you intend it to do?